### PR TITLE
Add get packagebundlecontroller command

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

File: cmd/eksctl-anywhere/cmd/getpackagebundlecontroller.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+func init() {
+	getCmd.AddCommand(getPackageBundleControllerCommand)
+	getPackageBundleControllerCommand.Flags().StringVarP(&gpbco.output, "output", "o", "", "Specifies the output format (valid option: json, yaml)")
+}
+
+var getPackageBundleControllerCommand = &cobra.Command{
+	Use:          "packagebundlecontroller(s) [flags]",
+	Aliases:      []string{"packagebundlecontroller", "packagebundlcontrolleres", "pbc"},
+	Short:        "Get packagebundlecontroller(s)",
+	Long:         "This command is used to display the currently packagebundlecontrollers",
```
Comment (ivyostosh) [inline cmd/eksctl-anywhere/cmd/getpackagebundlecontroller.go:22]: currently active packagebundlecontrollers?

Comment (TerryHowe) [reply cmd/eksctl-anywhere/cmd/getpackagebundlecontroller.go:22]:
```suggestion
	Long:         "This command is used to display the current packagebundlecontrollers",
```

**Comment by a-cool-train** [comment]
Why do we want to expose our internal workings to the client? I think packagebundlecontroller should not be exposed.

**Comment by TerryHowe** [comment]
> Why do we want to expose our internal workings to the client? I think packagebundlecontroller should not be exposed.

If the package controller runs on the management cluster, the PBC will be the only way to tell the active bundle. Each cluster could have a different active bundle.

**Review by a-cool-train — APPROVED** [review]

**Pull request merged**